lib/Core/Executor.cpp

```diff
@@ -2904,7 +2903,7 @@ void Executor::callExternalFunction(ExecutionState &state,
       if (i != arguments.size()-1)
 	os << ", ";
     }
-    os << ")";
+    os << ") at " << target->info->file << ":" << target->info->line;
```

Comment:
Hi @MartinNowack , I could use this, but I don't like the extra spaces it adds.  Should I perhaps reuse them and add them instead to where you call printFileLine in printDebugInstructions?